errors.go

```diff
@@ -27,24 +31,45 @@ func (e *Error) Is(target error) bool {
 	return e.Message == err.Message
 }
 
+var (
+	ErrPlaywright             = errors.New("playwright")
+	ErrTargetClosed           = errors.New("target closed")
+	ErrBrowserClosed          = errors.New("Browser has been closed")
+	ErrBrowserOrContextClosed = errors.New("Target page, context or browser has been closed")
```

Comment:
```suggestion
```
can't we remove these two? Ideally we only have ErrorPlaywright, ErrTimeout and ErrTargetClosed. (maybe use more Go like names, but something around these lines)

Reply:
[The latest commit](https://github.com/playwright-community/playwright-go/pull/394/commits/73f42d43db72e14a304572fa69d1d9fd8caedaa3) refactors the error type, bringing breaking changes:

- `ErrPlaywright` wraps all Playwright errors
- `ErrTimeout` wraps all timeout errors and replaces `TimeoutError`

pls review it.